From 97397ab47d8ae364b7a862957925d3d3314b188a Mon Sep 17 00:00:00 2001
From: Jeffy Chen <jeffy.chen@rock-chips.com>
Date: Mon, 30 Nov 2020 15:28:14 +0800
Subject: [PATCH 43/81] configure: Enable gles highp by default

Using mediump float precision for GLES would cause lots of display
issues.

Let's recommend highp precision by default.

Signed-off-by: Jeffy Chen <jeffy.chen@rock-chips.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index b5ea4b9b9..6121e4b7c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -584,7 +584,7 @@ dnl kdrive options
 AC_ARG_ENABLE(libunwind,      AS_HELP_STRING([--enable-libunwind], [Use libunwind for backtracing (default: auto)]), [LIBUNWIND="$enableval"], [LIBUNWIND="auto"])
 AC_ARG_ENABLE(xshmfence,      AS_HELP_STRING([--disable-xshmfence], [Disable xshmfence (default: auto)]), [XSHMFENCE="$enableval"], [XSHMFENCE="auto"])
 
-AC_ARG_ENABLE(gles-highp,    	      AS_HELP_STRING([--enable-gles-highp], [Use highp as default precision for GLES(default: no)]), [GLES_HIGHP=$enableval], [GLES_HIGHP=no])
+AC_ARG_ENABLE(gles-highp,    	      AS_HELP_STRING([--enable-gles-highp], [Use highp as default precision for GLES(default: yes)]), [GLES_HIGHP=$enableval], [GLES_HIGHP=yes])
 
 dnl chown/chmod to be setuid root as part of build
 dnl Replaces InstallXserverSetUID in imake
-- 
2.38.5

